### PR TITLE
add BaseConverter

### PR DESCRIPTION
**Pull request by lilia-project**

### CODE REVIEW

File: src/BaseConverter/BaseConverter.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package BaseConverter;
+
+import java.util.Scanner;
+
+public class BaseConverter {
+    public static void main(String[] args) {
+        do {
+            Scanner scanner = new Scanner(System.in);
+            System.out.println();
+            System.out.println("выберите число, соответствующее варианту конвертации");
```
Comment (edvardino) [inline src/BaseConverter/BaseConverter.java:10]: Just for the future, try not to use Cyrillic in your programs, only English. Developers that will be supporting your code will be thankful  :)

File: src/BaseConverter/BaseConverter.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+            if (option != 1 & option != 2 & option != 3 & option != 4) {
+
+                System.out.println("такой конвертер не предусмотрен");
+                return;
+            } else {
+                System.out.println("введите значение температуры в градусах Цельсия");
+            }
+            double tempCelsius = scanner.nextDouble();
+            Converter converter = new Converter();
+            converter.tempConverter(option, tempCelsius);
```
Comment (edvardino) [inline src/BaseConverter/BaseConverter.java:28]: give names to your methods using verbs. The method does smth. For instance, convert(tempreture), or transform(temp) etc.

File: src/BaseConverter/Converter.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
+package BaseConverter;
+
+public class Converter {
+    void tempConverter(int option, double tempCelsius) {
```
Comment (edvardino) [inline src/BaseConverter/Converter.java:4]: Really **nice** names for variables 💯 👍

**Review by edvardino — APPROVED** [review]

**Comment by edvardino** [comment]
Well done! You can merge it on your own, just take a look at a few comments for future 🥇

File: src/BaseConverter/BaseConverter.java (hunk 2 above)
Comment (lilia-project) [reply src/BaseConverter/BaseConverter.java:28]: it's clear

**Pull request merged**